app/scripts/modules/google/src/instance/custom/customInstanceBuilder.gce.service.js

```diff
@@ -10,57 +10,100 @@ export const name = GOOGLE_INSTANCE_CUSTOM_CUSTOMINSTANCEBUILDER_GCE_SERVICE; //
 module(GOOGLE_INSTANCE_CUSTOM_CUSTOMINSTANCEBUILDER_GCE_SERVICE, []).factory(
   'gceCustomInstanceBuilderService',
   function() {
-    function vCpuCountForLocationIsValid(vCpuCount, location, locationToInstanceTypesMap) {
-      const max = locationToInstanceTypesMap[location].vCpuMax;
+    function vCpuCountForLocationIsValid(instanceFamily, vCpuCount, location, locationToInstanceTypesMap) {
+      let max = 0;
+      switch (instanceFamily) {
+        case 'E2':
+          max = 32;
```

Comment:
Why is `E2` a special case here? Maybe we can include a comment explaining?

Reply:
e2 support up to 32 cores today
https://cloud.google.com/compute/docs/machine-types#e2_machine_types

Reply:
Thanks for clarifying! Would you mind including a link to that documentation inline in the code?

Reply:
Done!

Reply:
Thanks!